conf/keys/cfs/cfs-dev.sec

```diff
@@ -0,0 +1,2 @@
+Ga5I1u55+hH9kNKLFzztqBpKL0uI/IoAOg0jhwAwAWIpDCXmriFCPBDG74Fh+bgakWR2B0/aobX0
+o6a6/rvGMw==
```

Comment:
I don't know if this will cause problems but this line looks truncated

Reply:
um, works just fine for me